imports_test.go

```diff
@@ -221,17 +221,17 @@ func TestServiceImportWithWildcard(t *testing.T) {
 func TestImportsValidation(t *testing.T) {
 	ak := createAccountNKey(t)
 	akp := publicKey(ak, t)
-	i := &Import{Subject: "foo", Account: akp, To: "bar", Type: Stream}
-	i2 := &Import{Subject: "foo.*", Account: akp, To: "bar", Type: Service}
+	i := &Import{RemoteSubject: "foo", Account: akp, LocalSubject: "bar", Type: Stream}
+	i2 := &Import{RemoteSubject: "foo.*", Account: akp, LocalSubject: "bar", Type: Service}
 
 	imports := &Imports{}
 	imports.Add(i, i2)
 
 	vr := CreateValidationResults()
 	imports.Validate("", vr)
 
-	if len(vr.Issues) != 3 {
-		t.Errorf("imports without token or url should warn the caller x2, wildcard service as well")
+	if len(vr.Issues) != 2 {
```

Comment:
why did this change - did you change wildcard service? Derek wants to save that for later i think